src/lightning/app/core/app.py

```diff
@@ -69,6 +70,9 @@
 
 logger = Logger(__name__)
 
+if platform.system() == "Windows":
+    __doctest_skip__ = ["LightningApp.__init__"]
```

Comment:
Why not remove the `>>>` from the example so it doesn't get identified as a doctest?

Reply:
What is run is already tested extensively in the test suite. We don't need to run it as a doctest for the other platforms

Reply:
because it is tested with other OS and better have tested example

Reply:
It's already tested in all OS in the test suite. It doesn't cover anything that's not in the regular tests

Reply:
ok, then remove it completely

Reply:
Done. Pushed a commit

Reply:
no, you made it as example not removed the it so closing this PR

Reply:
I assume the original author of the docstring wanted to have it as an example, hence why I didn't remove it completely.

Since you closed this PR, do you want that I open one myself?

Reply:
Because it not tested we canot know if it works, so when you open new PR, I will request to be tested somehow...